sample_files/problem_setting/identical_checker_interactor.cpp

```diff
@@ -0,0 +1,152 @@
+#include <cstdio>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <regex.h>
```

Comment:
Actually wait... why are you using the POSIX regex library instead of the C++11 one?

Reply:
The C++11 library takes 5 seconds to compile on DMOJ, which is unacceptably slow for a checker/interactor. The C library compiles instantly, since it's available as a shared library instead of a template header library.

Reply:
Okay fine. I don't like the POSIX library, but C++ sucks.